examples/p4est_2d_dgsem/elixir_euler_NACA0012airfoil_mach085.jl

```diff
@@ -0,0 +1,133 @@
+# The outer boundary is constant but subsonic, so we cannot compute the
+# boundary flux for the external information alone. Thus, we use the numerical flux to distinguish
+# between inflow and outflow characteristics
+@inline function boundary_condition_subsonic_constant(u_inner,
+                                                      normal_direction::AbstractVector, x,
+                                                      t,
+                                                      surface_flux_function,
+                                                      equations::CompressibleEulerEquations2D)
+    u_boundary = initial_condition_mach085_flow(x, t, equations)
+
+    return Trixi.flux_hll(u_inner, u_boundary, normal_direction, equations)
+end
+
+boundary_conditions = Dict(:Left => boundary_condition_subsonic_constant,
+                           :Right => boundary_condition_subsonic_constant,
+                           :Top => boundary_condition_subsonic_constant,
+                           :Bottom => boundary_condition_subsonic_constant,
+                           :AirfoilBottom => boundary_condition_slip_wall,
+                           :AirfoilTop => boundary_condition_slip_wall)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
+                                    boundary_conditions = boundary_conditions)
+
+###############################################################################
+# ODE solvers
+
+# Run for a long time to reach a steady state
+tspan = (0.0, 20.0)
+ode = semidiscretize(semi, tspan)
+
+# Callbacks
+
+summary_callback = SummaryCallback()
+
+analysis_interval = 2000
+
+linf = 1.0 # Length of airfoil
+
+force_boundary_names = [:AirfoilBottom, :AirfoilTop]
+drag_coefficient = AnalysisSurfaceIntegral(semi, force_boundary_names,
+                                           DragCoefficientPressure(aoa(), rho_inf(),
+                                                                   U_inf(equations), linf))
+
+lift_coefficient = AnalysisSurfaceIntegral(semi, force_boundary_names,
+                                           LiftCoefficientPressure(aoa(), rho_inf(),
+                                                                   U_inf(equations), linf))
+
+analysis_callback = AnalysisCallback(semi, interval = analysis_interval,
+                                     output_directory = "out",
+                                     save_analysis = true,
+                                     analysis_integrals = (drag_coefficient,
+                                                           lift_coefficient))
+
+alive_callback = AliveCallback(analysis_interval = analysis_interval)
+
+save_solution = SaveSolutionCallback(interval = 500,
+                                     save_initial_solution = true,
+                                     save_final_solution = true,
+                                     solution_variables = cons2prim)
+
+# Small time step should be used to reach steady state
```

Comment:
I thought we needed both, although I do not have a good justification for it. Should we simply remove the comment?

Reply:
I agree with Andrew, I also do not see the point why you would need small timesteps - in fact, I think larger timesteps are beneficial as they bring you closer to "steady" state.

Reply:
> I think larger timesteps are beneficial as they bring you closer to "steady" state.

I could be wrong, but I think this will be true if the steady state is at a time $\approx T$ that we want to reach. In this test, it is also about doing a certain number of iterations (time steps) to reach the steady state. As evidence to that, note that local time stepping is often used in these simulations to reach steady state sooner.

Note that my point is only that there is no disadvantage in decreasing the time step size.

Reply:
Sorry, my wording was misleading - I meant that larger timesteps bring you in a finite time closer to a steady state as you just make more timesteps in fixed computational time.

Reply:
Time accuracy, i.e. small time steps, is not needed when the goal is to drive to steady-state. We just want to evolve the solution to steady-state as quickly as possible. However, global explicit time stepping always must obey the CFL constraint of the smallest elements to remain stable. So, the time step on the larger elements is taken small for this reason. That is, the mesh induces some stiffness in the problem. If the small CFL taken in this problem is to keep everything stable for the global time step, then we should update this comment accordingly.

Local time stepping (LTS) methods, as mentioned above, can mitigate this mesh stiffness by letting each element (or maybe groups of similarly sized elements) to run at their own locally stable explicit time step. Such LTS machinery is not implemented in Trixi and (as far as I know) nor in OrdinaryDiffEq.

Reply:
I just increased the CFL number to speed up the simulation.

Reply:
Okay, sounds good. Although this has opened up the possibility that adding LTS to remove mesh stiffness is a good Master's student project.